Gems/Streamer/StreamerProfiler/CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+
+# Add the Streamer.Static target
+ly_add_target(
+    NAME StreamerProfiler.Static STATIC
```

Comment:
Looks like you are not using the latest up-to-date Gem Template.
Can you re-create the gem using the latest DefaultGem template?